src/js/components/RangeSelector/stories/Basic.stories.js

```diff
@@ -2,7 +2,7 @@ import React, { useState } from 'react';
 
 import { Box, RangeSelector, Stack, Text } from 'grommet';
 
-export const Simple = ({ direction = 'horizontal' }) => {
+export const Basic = ({ direction = 'horizontal' }) => {
```

Comment:
why the name change here? was it just to match the file with naming?

Reply:
We already had a 'simple' story for RangeSelector so I just changed the name to 'Basic' to match the file name and so that we wouldn't have a duplicate

Reply:
gotcha makes sense